physics/MP/Morrison_Gettelman/aerinterp.F90

```diff
@@ -34,9 +35,395 @@ logical function netcdf_check(status, errmsg, errflg, why)
       endif
 
       END function netcdf_check
+!!!!!!!
+      SUBROUTINE read_aerdata_dl (me, master, iflip, idate, FHOUR, errmsg, errflg)
+      use machine, only: kind_phys, kind_io4,  kind_dbl_prec
+      use aerclm_def
+      use netcdf
+
+!--- in/out
+      integer, intent(in) :: me, master, iflip, idate(4)
+      character(len=*), intent(inout) :: errmsg
```

Comment:
Why are these declared as intent(inout) when they are immediately reset below?

Reply:
just copied from the old error handling code from read_aerdata. I do not know why and the code is added late by someone else.